include/sycl_ext_complex.hpp

```diff
@@ -558,6 +590,26 @@ class complex<_Tp, typename std::enable_if<is_genfloat<_Tp>::value>::type> {
     return __t;
   }
 
+#if defined(_SYCL_EXT_CPLX_FAST_MATH) &&                                       \
+    defined(_SYCL_EXT_CPLX_FAST_MATH_UNSAFE)
+  // This implementation is around 20% faster for single precision, 5% for double,
+  // at the expense of larger error in some cases, because no scaling is done.
+  // Only enable if the special unsafe macro is set and the compiler's fast
+  // math flag is set. Note that GNU libc++ uses this implementation, at least
```

Comment:
can you remove the GNU libc++ mention? This can cause some ugly licensing problems...